functions/lib/integration/import-product.js

```diff
@@ -175,7 +177,15 @@ module.exports = ({ appSdk, storeId, auth }, tinyToken, queueEntry, appData, can
               return null
             } else if (!product && tinyProduct && tipo === 'produto') {
               return parseProduct(tinyProduct, storeId, auth, true, tipo).then(product => {
-                return appSdk.apiRequest(storeId, '/products.json', 'POST', product, auth).then(response => {
+                return appSdk.apiRequest(storeId, '/products.json', 'POST', product, auth).then(async (response) => {
+                  if (appData.enable_category_import && tinyStockUpdate?.produto?.arvoreCategoria) {
+                    const { response: { data: { _id: newProductId } } } = response
+                    const { arvoreCategoria } = tinyStockUpdate?.produto
```

Comment:
```suggestion
                    const arvoreCategoria = tinyStockUpdate?.produto?.arvoreCategoria
```

Optional chaining não previne fatal com destruição de objeto

Reply:
Quer dizer, se `tinyStockUpdate?.produto` for undefined daria erro no seu código